openstack/imageservice/v2/images/requests.go

```diff
@@ -56,12 +69,52 @@ type ListOpts struct {
 
 	// SortDir will sort the list results either ascending or decending.
 	SortDir string `q:"sort_dir"`
-	Tag     string `q:"tag"`
+
+	// Tags filters on specific image tags.
+	Tags []string `q:"tag"`
+
+	// CreatedAt filters images based on their creation date.
+	CreatedAt *ImageDateQuery
```

Comment:
I wonder if these should be named differently since `CreatedAt` is used elsewhere as a `time.Time`. Would something like `CreatedAtQuery` be any better?

Reply:
Makes sense to me. `CreatedAtQuery` it is.